defaults/main.yml

```diff
@@ -14,6 +14,7 @@ LOAD_BALANCER_CERTS_DIR: /etc/haproxy/certs
 LOAD_BALANCER_CONF_DIR: /etc/haproxy/conf.d
 LOAD_BALANCER_BACKENDS_DIR: /etc/haproxy/backends
 LETSENCRYPT_ARCHIVE_DIR: /etc/letsencrypt/archive
+LETSENCRYPT_LIVE_DIR: /etc/letsencrypt/live
```

Comment:
@pomegranited I'm already using it everywhere except in `manage_certs.py`.  That file currently hard-codes all letsencrypt paths, since template variable expansions in the middle of a Python file confuses pylint and makes running the tests locally impossible.  All configuration flags are passed in via the configuration file.  It's reasonable to add command-line flags to configure the letsencrypt paths, but I think this is out of scope for the current fix.